README.rst

```diff
@@ -474,6 +474,19 @@ A lower-level Faidx class is also available:
    where "filename.fa" is the original FASTA file.
 -  Start and end coordinates are 1-based.
 
+Support for compressed FASTA
```

Comment:
I like the section you've added here. It definitely belongs higher up in the README, so I'll move a few sections around to add the more useful features near the top.